### DISCUSSION TITLE
Do we need a timeline component?

### DISCUSSION THREAD

**Discussion by wa-rren-dev** [nhsevidence/NICE-Experience#135] [Ideas]
I've found it in three places thus far:

- https://www.nice.org.uk/about/what-we-do/life-sciences/scientific-advice/express-scientific-advice
- https://www.nice.org.uk/about/what-we-do/life-sciences/scientific-advice
- https://www.nice.org.uk/about/what-we-do/life-sciences/scientific-advice/eunethta-multi-hta-advice

**Comment by rnickr** [comment]
Bumping this. We should look at creating a timeline component for the Design System. Comms have raised issues with the current component which need looking at.
